akka-docs-dev/rst/scala/code/docs/stream/IntegrationDocSpec.scala

```diff
@@ -177,9 +178,9 @@ class IntegrationDocSpec extends AkkaSpec(IntegrationDocSpec.config) {
     import Supervision.resumingDecider
 
     val emailAddresses: Source[String, Unit] =
-      authors.section(supervisionStrategy(resumingDecider)) {
-        _.mapAsync(author => addressSystem.lookupEmail(author.handle))
-      }
+      authors.via(
+        Flow[Author].mapAsync(author => addressSystem.lookupEmail(author.handle))
```

Comment:
Unrelated: Was wondering if we could make `.via(_.mapAsync{})` constructs work...